bindings/python/requirements.txt

```diff
@@ -1,3 +1,3 @@
 # Apache License 2.0
 
-open-space-toolkit-core~=2.1.0
+open-space-toolkit-core~=2.1
```

Comment:
I don't know if this is a good idea to do. 

This will let the Python version get auto-bumped to 2.2, 2.3, etc. Since we only allow the CPP dependencies to bump patch versions, this could get us out of sync with CPP.

Reply:
Gotcha, so then we should allow all to be patch auto-updates